charts/consul/templates/client-daemonset.yaml

```diff
@@ -234,8 +234,11 @@ spec:
             {{- if .Values.global.tls.enableAutoEncrypt }}
             - name: CONSUL_HTTP_SSL_VERIFY
               value: "false"
-            {{- else }}
+            {{- end }}
```

Comment:
I'm curious why this was switched to an `end` instead of an `else`? With auto-encrypt, we still want to skip ssl verification because the CA of the client != the CA of the server.

Reply:
Oh and since vault only works with auto-encrypt, we don't need this change for the client then! Sorry for confusing it in the task; I only realized this after looking at your PR!

Reply:
No problem. So just reset client-daemonset back to what is on main?

Reply:
yeah, we don't need to change the client